src/www/services_ntpd.php

```diff
@@ -425,7 +425,7 @@ function removeRow() {
                       <input name="nomodify" type="checkbox" id="nomodify"<?=empty($pconfig['nomodify']) ? " checked=\"checked\"" : ""; ?> />
                       <?=gettext("Deny state modifications (i.e. run time configuration) by ntpq and ntpdc"); ?>
                       <br />
-                      <input name="noquery" type="checkbox" id="noquery"<?=!empty($pconfig['noquery']) ? " checked=\"checked\"" : ""; ?> />
+                      <input name="noquery" type="checkbox" id="noquery"<?=empty($pconfig['noquery']) ? " checked=\"checked\"" : ""; ?> />
```

Comment:
hmm I don't think we should reverse this? WRT the config render

Reply:
These go along with the `// really stupid fields usage`  - if your head already hurts, I'm not surprised. Without this additional commit which I originally missed, it does not work properly.